client/reader/discover/style.scss

```diff
@@ -32,3 +35,30 @@
 		pointer-events: none;
 	}
 }
+
+.reader-discover-login-prompt {
+	text-align: center;
+	max-width: 380px;
+	margin: 0 auto;
+
+	h2 {
+		color: var(--color-neutral-70);
+		font-size: 1.25rem;
+		font-weight: 600;
+		margin-bottom: 9px;
+	}
+
+	p {
+		font-size: 1rem;
+		margin-bottom: 16px;
+	}
+
+	.components-button {
+		display: block;
+		margin: 0 auto;
+	}
+
+	.reader-discover-login-prompt__signup-button {
+		margin-bottom: 16px;
+	}
```

Comment:
In theory, `&__signup-button` should work here, but in practice, it does not.

Reply:
I like full names 😄 . They are easier to search.

Reply:
> I like full names 😄 . They are easier to search.

Huge +1 about the searching.